packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -257,7 +257,7 @@ export class SDKClient {
       fileId = await this.createFile(
         ethereumTransactionData.callData,
         this.clientMain,
-        requestId,
+        requestId ?? 'create-file-from-submit-ethereum-transaction',
```

Comment:
What's the goal of this?

Reply:
The optional parameter on the requestId causes `Argument of type 'string | undefined' is not assignable to parameter of type 'string'.`. I can remove the optional all the way to the `sendRawTransaction`

Reply:
My concern is the choice of string.
Doesn't seem consistent with anything we've done.
Either the requestId is provided by a user of one is generated higher up if not present.

Why do we need this workaround only here?